frappe/public/js/frappe/ui/dialog.js

```diff
@@ -216,6 +216,7 @@ frappe.ui.Dialog = class Dialog extends frappe.ui.FieldGroup {
 	hide() {
 		this.$wrapper.modal("hide");
 		this.is_visible = false;
+		frappe.ui.hide_open_dialog();
```

Comment:
```suggestion
```
hide command of one dialog should not affect other dialogs. There might be a valid use case to keep multiple dialogs open... (other dialogs might be in the minimized state)